test/pages.js

```diff
@@ -457,6 +457,79 @@ describe('Pages', function() {
     assert.strictEqual(page.path, `${homeId.replace(':en:published', '')}/another-parent/parent/sibling`);
   });
 
+  it('moving peer /parent/peer-page into /parent should ends with /parent/peer-page', async function() {
```

Comment:
Please add coverage of the more common case as well, e.g. where the peers have slugs `/a` and `/b` and the outcome should be `/a/b`. Looking quickly over the existing tests I am not convinced we have that covered right now.